benchmarks/asv.conf.json

```diff
@@ -6,6 +6,10 @@
     "environment_type": "conda-delegated",
     "show_commit_url": "https://github.com/scitools/iris/commit/",
     "branches": ["upstream/main"],
+    "build_command": [
+        "python setup.py build",
+        "python -mpip wheel --no-deps -w {build_cache_dir} {build_dir}"
```

Comment:
[pip wheel docs](https://pip.pypa.io/en/stable/cli/pip_wheel/)

`--no-deps`: as with all of Iris we want the Conda environment management to take care of dependencies, without Pip making further changes.

`{build_cache_dir}` and `{build_dir}` - can see how these are used from various examples [in the ASV documentation](https://asv.readthedocs.io/en/v0.6.3/asv.conf.json.html#build-command-install-command-uninstall-command).

ASV sometimes also uses `PIP_NO_BUILD_ISOLATION=false` and `--no-index` to account for benchmarks being run without Internet access, but that is not a concern for us.